app/backend/src/couchers/tasks.py

```diff
@@ -30,6 +30,7 @@ def send_login_email(user, token, expiry_text):
 def send_password_reset_email(user, token, expiry_text):
     logger.info(f"Sending password reset email to {user=}:")
     password_reset_link = urls.password_reset_link(password_reset_token=token.token)
+    logger.info(f"Link is: {password_reset_link}")
```

Comment:
Small modification on the backend so I can actually see the reset password link in the console in dev mode.